luxonis_ml/data/augmentations/custom/mixup.py

```diff
@@ -101,7 +101,7 @@ def apply_to_mask_batch(
         return out_mask
 
     @override
-    def apply_to_bboxes_batch(
+    def apply_to_bboxes(
```

Comment:
Is now batched augmentation transformed to the point that it can call the same API as the normal (1 image) augmentations? As I see most of the batch reated code got removed and also this now overrides the methods of single image augmentations.

Reply:
Yeah, it uses the same API now. Most of the batch related code wasn't necessary for our usage.